crates/astria-core/CHANGELOG.md

```diff
@@ -47,6 +50,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Replaced all instances of `[u8; 32]` by newtype
   `astria_core::sequencerblock::v1::block::Hash` where appropriate [#1884](https://github.com/astriaorg/astria/pull/1884).
 - Replace `execution::v1` with `execution::v2` API [#2006](https://github.com/astriaorg/astria/pull/2006).
+- Change `ExpandedBlockData::user_submitted_transactions` to UN-parsed
```

Comment:
This needs a section that `TRANSACTION_FAILED` was renamed to `TRANSACTION_FAILED_EXECUTION`

Reply:
Done in [2270e90](https://github.com/astriaorg/astria/pull/2142/commits/2270e90cc6f02a75e4ce0ea377e084f7031cace3).